contracts/tests/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+    "extends": "../../tsconfig",
+    "compilerOptions": { "outDir": "lib", "rootDir": ".", "resolveJsonModule": true },
+    "include": ["./src/**/*", "./test/**/*", "./generated-wrappers/**/*"]
```

Comment:
Should this `exclude` `./lib`? It seems like that fixed some issues that I had using `yarn build`